package.json

```diff
@@ -214,7 +214,7 @@
     "react-native-reanimated": "2.4.1",
     "react-native-redash": "16.2.2",
     "react-native-restart": "0.0.22",
-    "react-native-safe-area-context": "0.5.0",
+    "react-native-safe-area-context": "4.3.1",
```

Comment:
was this on purpose? seems like a big version bump

Reply:
this gives better inset calculations for android devices, specifically with soft menus.  otherwise should be a no op